src/WinGet.RestSource/Models/Schemas/Installer.cs

```diff
@@ -165,6 +166,48 @@ public Installer(Installer installer)
         /// </summary>
         public RestrictedCapabilities RestrictedCapabilities { get; set; }
 
+        /// <summary>
+        /// Gets or sets MSStoreProductIdentifier.
+        /// </summary>
+        [MSStoreProductIdentifierValidator]
+        public string MSStoreProductIdentifier { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether InstallerAbortsTerminal is set.
+        /// </summary>

```

Comment:
what does installer aborting terminal mean?

Reply:
Not sure about this. I can ask Demitrius and get back to you.

Reply:
Ok I got an answer from Yao. This bit tells if the shell apps abort terminal during Installation. Winget receives Ctrl-C signal when terminal exits and cancels operation. This is not yet used in WinGet client but will be used in future.

Reply:
Should there be more documentation about what this is here/in the schema?